GITHUB_SETUP.md

```
@@ -0,0 +1,285 @@
+Then add the secrets:
+
+```bash
+# AI provider key (match ai-provider)
+gh secret set ANTHROPIC_API_KEY --body "sk-ant-..."
+# or: OPENAI_API_KEY / GOOGLE_AI_API_KEY
+
+# GitHub PAT for the hosted MCP endpoint (repo / pull_requests scope)
+gh secret set YAMA_GITHUB_TOKEN --body "github_pat_..."
+```
+
+See [§6 Authentication](#6-authentication-deep-dive) for why a PAT (not the default
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)